components/api/src/main/java/com/hotels/styx/api/HttpResponse.java

```diff
@@ -98,43 +129,115 @@ public static Builder response(HttpResponseStatus status, StyxObservable<ByteBuf
         return new Builder(status).body(body);
     }
 
+    /**
+     * Returns a HTTP header value.
+     * <p>
+     * When the header has been set multiple times, this still returns
+     * just one value.
+     *
+     * @param name header name
+     * @return An optional of value, or empty if not present.
+     */
     @Override
     public Optional<String> header(CharSequence name) {
         return headers.get(name);
     }
 
+    /**
+     * Returns all HTTP headers as {@link HttpHeaders} instance.
```

Comment:
Should be "Returns all HTTP headers as {@link HttpHeaders} instance**s**".

Reply:
Why? It returns just a single `HttpHeaders` instance.